Sensus.Shared/Protocol.cs

```diff
@@ -400,6 +400,7 @@ public static void RunUnitTestingProtocol(Stream unitTestingProtocolFile)
         private bool _startImmediately;
         private DateTime _endTimestamp;
         private bool _continueIndefinitely;
+        private readonly List<Window> _notificationSoundExclusionWindows;
```

Comment:
It's not just sound. We should also exclude vibration. How about `_notificationAlertExclusionWindows`?